packages/tracing/src/browser/metrics.ts

```diff
@@ -362,21 +362,24 @@ export function addResourceSpans(
 }
 
 /** Create performance navigation related spans */
-function addPerformanceNavigationTiming(
-  transaction: Transaction,
-  entry: Record<string, any>,
-  event: string,
-  timeOrigin: number,
-  eventEnd?: string,
-): void {
+function addPerformanceNavigationTiming(props: {
+  transaction: Transaction;
+  entry: Record<string, any>;
+  event: string;
+  timeOrigin: number;
+  eventEnd?: string;
+  description?: string;
+}): void {
+  const { transaction, entry, event, timeOrigin, eventEnd, description } = props;
```

Comment:
Does it pay-off moving all args into a single object just to deconstruct it here?!

Reply:
Oh well, one advantage is being able to name an argument and omit others. 👍